lib/commons/color/get-background-color.js

```diff
@@ -44,16 +45,27 @@ function getBgColor(elm, elmStyle) {
 	return bgColor;
 }
 
-function calculateObscuringAlpha(elmIndex, elmStack) {
+function contentOverlapped(targetElement) {
```

Comment:
This function is confusing to me:

1. It has one argument, but is called with two arguments.
1. It is only called when there is an element above the target element but it is trying to find the target element using document.elementFromPoint - which should return the topmost element - which will therefore never be the target element

I don't think this does what it is intended to do